### ISSUE TITLE
Thumb preview Lora (same cannot change)

### ISSUE THREAD

**Issue by Khampol** [hayden-fr/ComfyUI-Model-Manager#31]
![image](https://github.com/user-attachments/assets/d2910be7-7fa1-4150-87de-af5dfacd74fa)

I DL Lora from Civit, all same image Thumb. An idea?

eg. from here : https://civitai.com/models/486893?modelVersionId=541632

**Comment by cdb-boop** [comment]
Yeah, I noticed this started happening a few days ago. Happens with all models.

Currently, the image url is parsed, and if it is a Civitai url, the model manager asks their API for the original image. For some reason, every request returns this image. This search should probably be disabled since it has had a high fail rate in the past, either because their API doesn't work or I'm using it wrong.

**Comment by Khampol** [comment]
So... I should install the dev fork instead? correct ?

**Comment by cdb-boop** [comment]
Yes, you might be right. My current work around was to remove "width=" and width value from the url.
